web/server.ts

```diff
@@ -93,6 +93,7 @@ export const server = (async function setupWebserver() {
     app.use('/api/attachments', attachmentRouter);
     app.use('/api/certificate', certificateRouter);
     app.use('/api/files', fileRouter);
+    app.use('/api', chatNotificationRouter);
```

Comment:
Could we move the router to it's own path, like '/api/chat/'? Feels a bit weird to delegate the whole api route to the router.